build.gradle

```diff
@@ -36,9 +36,12 @@ dependencies {
     compile("org.springframework.boot:spring-boot-starter-web")
     compile("org.springframework.boot:spring-boot-starter-mail")
     compile("org.springframework.boot:spring-boot-starter-thymeleaf")
-    compile("org.hsqldb:hsqldb")
+    compile("org.springframework.security:spring-security-test:4.0.2.RELEASE")
+    runtime("org.hsqldb:hsqldb")
```

Comment:
Er wordt een NotYetImplemented class gebruikt, die zit in de commons-lang3. Deze package zit er niet standard in, dus hierbij de import.
compile("org.apache.commons:commons-lang3:3.4")